src/overlays/actors/ovl_En_Invadepoh/z_en_invadepoh.c

```diff
@@ -20,492 +239,4494 @@ const ActorInit En_Invadepoh_InitVars = {
+    EnInvadepoh_InitRomani, EnInvadepoh_InitRomani, EnInvadepoh_InitRomani, EnInvadepoh_InitDog, EnInvadepoh_InitCremia, EnInvadepoh_InitRomani, func_80B46414,
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E1B0.asm")
+static EnInvadepohDestroyFunc D_80B4ECE8[] = {
+    func_80B46BB0, func_80B46BC0, func_80B46C08, func_80B46C34, func_80B46C50, func_80B46C50, func_80B46C7C,
+    func_80B46C50, func_80B46C50, func_80B46C50, func_80B46C94, func_80B46CC0, func_80B46CF4, func_80B46D28,
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E200.asm")
+static s16 D_80B4ED20[] = {
+    0x0082, 0x007D, 0x0073, 0x0064, 0x0050, 0x004E, 0x004C, 0x004A,
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E2AC.asm")
+static Vec3f D_80B4ED30[] = {
+    { 0.01f, 0.01f, 0.01f }, { 0.02f, 0.01f, 0.005f }, { -0.01f, 0.0f, 0.0f },
+    { 0.01f, 0.01f, 0.01f }, { 0.005f, 0.01f, 0.02f },
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E324.asm")
+static Vec3f D_80B4ED6C[] = {
+    { 0.0005f, 0.0279999990016f, 0.01f }, { -0.01f, 0.0f, 0.0f }, { -0.01f, 0.0f, 0.0f },
+    { 0.016f, 0.0004f, 0.01f },           { -0.01f, 0.0f, 0.0f }, { 0.0005f, 0.0005f, 0.0005f },
+    { 0.0002f, 0.0002f, 0.0002f },
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E3F0.asm")
+static s16 D_80B4EDC0[] = { 0xE0C0, 0xE890, 0xD508 };
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E5B0.asm")
+static s16 D_80B4EDC8[] = {
+    0xF8F8,
+    0xFC18,
+    0x0000,
+    0x07D0,
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E61C.asm")
+static Vec3f D_80B4EDD0[] = {
+    { -1813.0f, 374.0f, 1900.0f }, { 2198.0f, 153.0f, 3365.0f }, { -1434.0f, 262.0f, 3365.0f },
+    { -393.0f, 396.0f, 1084.0f },  { 0.0f, 1500.0f, 0.0f },
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E660.asm")
+static unkstruct_80B4EE0C D_80B4EE0C[] = {
+    { 0.08f, 0x02BC, 0xFFF6 },
+    { 0.09f, 0x012C, 0xFFFB },
+    { 0.05f, 0x0190, 0x0000 },
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E6E4.asm")
+static Vec3f D_80B4EE24 = {
+    2000.0f,
+    1000.0f,
+    0.0f,
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E784.asm")
+static Vec3f D_80B4EE30 = {
+    400.0f,
+    270.0f,
+    0.0f,
+};
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Invadepoh_0x80B439B0/func_80B4E7BC.asm")
+MtxF D_80B502A0;
+MtxF D_80B502E0;
+Actor* D_80B50320[0x8];
+u8 D_80B50340[0x8];
+UNK_TYPE1 D_80B50348;
+unkStruct_80B50350 D_80B50350[10];
+Actor* D_80B503F0;
+EnInvadepoh* D_80B503F4;
+Actor* D_80B503F8;
+AnimatedMaterial* D_80B503FC;
+AnimatedMaterial* D_80B50400;
+s16 D_80B50404[3];
+Actor* D_80B5040C;
```

Comment:
From what I saw, those `Actor*` variables seem to be `EnInvadepoh`, so you could probably change them.